src/gp_instance.cpp

```diff
@@ -174,3 +175,63 @@ void GPInstance::EstimateSBNParameters() {
   double marginal_log_lik = GetEngine()->GetLogMarginalLikelihood();
   std::cout << std::setprecision(9) << marginal_log_lik << std::endl;
 }
+
+RootedTreeCollection GPInstance::GetRootedTreeCollection() {
```

Comment:
I suggest "GenerateCompleteRootedTreeCollection" or "GenerateAllTrees". In particular, I like to reserve "Get" for accessors.

Reply:
Done.